src/libravi/RaviSession.ts

```diff
@@ -472,7 +472,7 @@ export class RaviSession {
   * @internal
  * Constants used as the default filter for the stats collected in the RaviStatsWatcher
  */
-const STATS_WATCHER_FILTER = new Map([
+export const STATS_WATCHER_FILTER = new Map([
```

Comment:
make it available for other code (like mine) to modify.

Reply:
"const" is a misleading name. It has come to mean "the _value_ here, which may be a container, will not change identity in the === sense".  Some languages have an additional term, such as "sealed" or "frozen", to mean "the _content_ here will not change in the == (or mumbleEqual) sense".  There are yet more possibilities, such as "final" to indicate a definition will not be extended (such as by subclassing).  The potential definition space is as big (or bigger?) as the for definitions of equality. (Common Lisp having ==, eq, eql, equal, and equalp.)